client/verta/tests/test_cli/test_endpoint.py

```diff
@@ -95,25 +95,25 @@ def test_get(self, client, created_entities, experiment_run, model_for_deploymen
             ["deployment", "get", "endpoint", path],
         )
         assert not result.exception
-        assert "path: {}".format(endpoint.path) in result.output
-        assert "id: {}".format(endpoint.id) in result.output
-        assert "curl: <endpoint not deployed>" in result.output
-
-        assert "status" in result.output
-        assert "date created" in result.output
-        assert "date updated" in result.output
-        assert "stage's date created" in result.output
-        assert "stage's date updated" in result.output
-        assert "components" in result.output
-
-        updated_status = endpoint.update(experiment_run, DirectUpdateStrategy(), True)
+
+        def in_output(s: str) -> bool:
+            return any(line.startswith(s) for line in result.output.splitlines())
+
+        assert in_output(f"path: {endpoint.path}")
+        assert in_output(f"id: {endpoint.id}")
+        assert in_output("curl: <endpoint not deployed>")
+        assert in_output("status")
+        assert in_output("date created")
+        assert in_output("stage's date created")
+        assert in_output("stage's date updated")
+        assert in_output("components")
```

Comment:
In this test (and the other test), I'm just

- defining a closure to check the output using `str.startswith()` instead of `in`
- using f-strings
- removing `"date updated"